cmd/cluster/support/delete_test.go

```diff
@@ -0,0 +1,39 @@
+package support
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestCreateDeleteRequest(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	testCases := []struct {
+		title          string
+		clusterID      string
+		lmtSprReasonID string
+		client         *Client
+		errExpected    bool
+	}{
+		{
+			title:          "Delete Request creation",
+			clusterID:      "5a5a5a5a-5a5a-5a5a-5a5a-5a5a5a5a5a5a",
+			lmtSprReasonID: "FAFAFAFA-FAFAFAFA-FAFAFAFA-FAFAFAFA",
```

Comment:
Do these match the pattern that the API uses? Can you show an example?

Reply:
Yes they do. But to make sure I added a validating function with a regEx, in order to make sure they match that allowed me to create a test that fails when i submit a bad ID. To be sumited with other changes shortly